### ISSUE TITLE
latest owncloud needs PHP 5.4.0

### ISSUE THREAD

**Issue by JasonBoyles** [onddo/owncloud-cookbook#19]
The latest owncloud wants PHP 5.4.0. This cookbook installs 5.3.10 on Ubuntu 12.04.

If you use the default recipe with the latest you'll only see this on page load:

```
This version of ownCloud requires at least PHP 5.4.0
<br/>
You are currently running 5.3.10-1ubuntu3.16. Please update your PHP version.
```

**Comment by raulr** [comment]
Thanks for reporting. I've fixed it on master.

**Comment by zuazo** [comment]
Fix released in [0.5.0](https://github.com/onddo/owncloud-cookbook/releases/tag/0.5.0).
